translations/monero-core_es.ts

```diff
@@ -459,7 +459,7 @@
     <message>
         <location filename="../LeftPanel.qml" line="439"/>
         <source>Prove/check</source>
-        <translation type="unfinished"></translation>
+        <translation>Probar o verificar</translation>
```

Comment:
Probar/verificar

Puntuaciones deben ser iguales, Hay que seguir [las guias :)](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/translating-the-gui)

Reply:
Oops. La inercia.